libindy/src/errors/pool.rs

```diff
@@ -81,6 +85,7 @@ impl ToErrorCode for PoolError {
             PoolError::Terminate => ErrorCode::PoolLedgerTerminated,
             PoolError::Timeout => ErrorCode::PoolLedgerTimeout,
             PoolError::AlreadyExists(_) => ErrorCode::PoolLedgerConfigAlreadyExistsError,
+            PoolError::InvalidCacheCleared => ErrorCode::PoolLedgerInvalidCache,
```

Comment:
If it's internal error, may be we should not pass it into API.
@vimmerru sounds like we have first case when some error is actually internal. May be we can map this case to CommonInvalidState or even to `unreachable` macros